src/components/nav/nav.scss

```diff
@@ -0,0 +1,55 @@
+body {
+  margin: 0;
+}
+
+nav {
+  display: flex;
+  width: 160px;
+  height: 1000px;
+  color: #fff;
+  background-color: RGBA(33,37,41,var(--bs-bg-opacity,1));
+  margin: 0;
+  padding: 1rem;
+
+  ul {
+    display: flex;
+    flex-direction: column;
+    list-style-type: none;
+    padding-left: 0;
+    width: -webkit-fill-available;
+    margin: 0;
+    padding: 0;
+
+    li {
+      display: flex;
+      flex-direction: column;
+      align-items: flex-start;
+      padding: 0.5rem;
```

Comment:
It's better to avoid mix of rem and px spacings. It should be rem or px, not both. In out case just update all rems to pixels.